etc/profile-m-z/signal-desktop.profile

```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
 netfilter
 nodvd
 nogroups
+noroot
```

Comment:
AFAIK noroot break electron/chromium based program in newer versions (see #2946).